src/lib.rs

```diff
@@ -415,6 +433,15 @@ pub async fn collect_from_metrics(
             context: "list podmetrics, maybe Metrics API not available".to_string(),
             source,
         })?;
+    compute_utilizations_metrics(pod_metrics, resources).await?;
+    Ok(())
+}
+
+#[instrument(skip(pod_metrics, resources))]
+pub async fn compute_utilizations_metrics(
+    pod_metrics: ObjectList<metrics::PodMetrics>,
```

Comment:
in this case it's utilization ;=), so maybe just align to other names (verb, formulation)